app/views/spree/users/_skinny.html.haml

```diff
@@ -2,7 +2,8 @@
   .columns.small-2
     %img.margin-top.account-logo{"logo-fallback" => true, "ng-src" => "{{shop.logo}}"}
   .columns.small-5
-    %h3.margin-top{"ng-bind" => "::shop.name"}
+    %h3.margin-top
+      %a{"ng-href" => "{{::shop.hash}}#{main_app.shop_path}", "ng-bind" => "::shop.name"}
```

Comment:
What purpose is `{{::shop.hash}}` serving here? 

Does `"ng-href" => "{{::shop.path}}"` work here, in place of `"ng-href" => "{{::shop.hash}}#{main_app.shop_path}"`?

Reply:
I will check that, thanks for reviewing. On the Orders page, I had to do that way to work.

Reply:
It did not work!